src/NuGet.Core/NuGet.Packaging/Strings.resx

```diff
@@ -149,8 +149,9 @@
   <data name="MinClientVersionAlreadyExist" xml:space="preserve">
     <value>MinClientVersion already exists in packages.config</value>
   </data>
-  <data name="MissingNuspec" xml:space="preserve">
-    <value>Nuspec file does not exist in package.</value>
+  <data name="Error_MissingNuspecFile" xml:space="preserve">
+    <value>The package doesn't contain .nuspec file. {0}</value>
```

Comment:
How about: 

> The package is missing the required nuspec file. 

Not sure if we use .nuspec or just nuspec throughout our error messages, I'd suggest you review that before settling on a style.